models/api_rate.go

```diff
@@ -62,18 +62,25 @@ func (r *Rate) SpecialServicesRequested() *SpecialServicesRequested {
 	}
 }
 
+func maximumBetween(x float64, y float64) float64 {
```

Comment:
The function name is maximumBetween while it returns min between, should be a typo